quinn/src/connection.rs

```diff
@@ -280,6 +280,10 @@ impl Future for ConnectionDriver {
 /// automatically closed with an `error_code` of 0 and an empty `reason`. You can also close the
 /// connection explicitly by calling [`Connection::close()`].
 ///
+/// Closing the connection will make both peers immediately drop all the stream data not yet
+/// delivered to the application. [`Connection::close()`] describes in more detail how to
```

Comment:
This claim is too strong: Quinn currently does, and other conforming implementations may, allow previously received data to be read after a connection is closed, although as we discussed this is not safe to rely on. How about something like:

> Closing the connection will immediately abandon efforts to deliver data to the peer.